servicecontrol/creating-config-file.md

```diff
@@ -257,6 +257,9 @@ Type: string
 
 Default: `<ErrorQueue>.log`
 
+Version 1.28 and below created the queue specified by this setting when the service instance is installed regardless of the value of `ServiceControl/ForwardErrorMessages`.  From V1.29 the queue is only created if `ServiceControl/ForwardErrorMessages` is enabled. 
```

Comment:
>From V1.29 the queue is only created if `ServiceControl/ForwardErrorMessages` is enabled. 

Should be `When using Versions 1.29 and above, ...`
See Version language here: 
https://github.com/particular/docs.particular.net/#version-language